src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
With our current agreement about how the UI framework should work, we can not have a wrapping using dividers. Components should come with their renderers for their own namespace, that just renders the components in that namespace and nothing else. So no, Dividers do not allow accordions or we need to have listing dividers and dropdown dividers, or we need to revise our rather carefully designed rules, or we drop namespaces.

In general I can live with a lot of things, but I really need to point out consequences and tradeoffs =)